neural_clbf/controllers/clf_controller.py

```diff
@@ -40,13 +41,17 @@ def __init__(
             clf_lambda: convergence rate for the CLF
             clf_relaxation_penalty: the penalty for relaxing CLF conditions.
             controller_period: the timestep to use in simulating forward Vdot
+            disable_gurobi: if True, gurobi will not be used
```

Comment:
"gurobi will not be used" -> "Gurobi will not be used during evaluation. Default is train with CVXPYLayers, evaluate with Gurobi; setting this to true will evaluate with CVXPYLayers instead (to avoid requiring a Gurobi license)."

Reply:
see commit [2b05d3e](https://github.com/MIT-REALM/neural_clbf/pull/9/commits/2b05d3e9f2d8a94e18d0b5cb6f31c73a5e3cf4e2)